benches/bench_basic.rs

```diff
@@ -2,6 +2,12 @@ extern crate redis;
 #[macro_use]
 extern crate bencher;
 
+extern crate futures;
+extern crate tokio_core;
+
+use futures::Future;
+use tokio_core::reactor::Core;
```

Comment:
`tokio_core` is not in the dependencies anymore. This whole benchmark needs a rewrite.